README.md

```diff
@@ -182,6 +182,7 @@ LMDeploy uses AWQ algorithm for model weight quantization
 
 ```
 python3 -m lmdeploy.lite.apis.auto_awq \
+  --model $HF_MODEL \
   --w_bits 4 \                       # Bit number for weight quantization
   --w_sym False \                    # Whether to use symmetric quantization for weights
```

Comment:
Hide this line for now, I forgot to change it earlier. There might be some issues with w_sym = True during Turbomind inference currently.